### ISSUE TITLE
Login works but import don´t find users..

### ISSUE THREAD

**Issue by Albvadi** [Adldap2/Adldap2-Laravel#393]
- Laravel Version: 5.5.4
- Adldap2-Laravel Version: 7.0.15
- PHP Version: 7.0.9

### Description:
Hi, 
the login from the front-end works well. If the user doesn´t exists in the DB, adldap2, sync the user and create in the db and login correctly.
But, the import command, no found users. 

I tried with different commands but always found 0 users:
* `php artisan adldap:import user.valid`
* `php artisan adldap:import --filter "(uid=user.valid)"`
* `php artisan adldap:import --filter "uid=user.valid"`
* `php artisan adldap:import --connection ldap "(uid=user.valid)"`

Nothing it´s written in the log...
Any idea??

### Steps To Reproduce:
This is my config:

adldap_auth.php
```PHP
return [
    'connection' => env('ADLDAP_CONNECTION', 'ldap'),
    'provider' => Adldap\Laravel\Auth\DatabaseUserProvider::class,
    'resolver' => Adldap\Laravel\Auth\Resolver::class,
    'importer' => Adldap\Laravel\Auth\Importer::class,
    'rules' => [
        //Only valid users from company
        App\Rules\Adldap\ldapFilter::class,
    ],
    'scopes' => [
        // Only allows users with a user principal name to authenticate.
        // Adldap\Laravel\Scopes\UpnScope::class,
    ],
    'usernames' => [
        'ldap' => 'uid', // Changed from 'userprincipalname'
        'eloquent' => 'username', // Changed from 'userprincipalname'
    ],
    'login_fallback' => env('ADLDAP_LOGIN_FALLBACK', false),
    'password_sync' => env('ADLDAP_PASSWORD_SYNC', true),
    'windows_auth_attribute' => ['samaccountname' => 'AUTH_USER'],
    'sync_attributes' => [
        'username' => 'uid',
        'email' => 'mail',
        'name' => 'cn',
    ],
];
```
Rule: ldapFilter
```PHP
namespace App\Rules\Adldap;
use Adldap\Laravel\Validation\Rules\Rule;

class ldapFilter extends Rule
{
    public function isValid()
    {
    	$validUsers = [
    		'user.valid',
    	];

    	$validOus = [
            'ou=dc,ou=company,o=com',
        ];

        return in_array($this->user->getFirstAttribute('uid'), $validUsers) || str_contains($this->user->getDN(), $validOus);
    }
}
```

**Comment by JustJeremy** [comment]
If you leave off the custom rules are you able to search for a user and import from command line?

**Comment by Albvadi** [comment]
> If you leave off the custom rules are you able to search for a user and import from command line?

Nop

**Comment by Albvadi** [comment]
I have 2 connections configured:

- ldap (basedn = `o=intranet.rts`)
- adcare (basedn = `DC=company,DC=com`)

Command: `php artisan adldap:import  --connection "ldap" "(uid=user.testing)"`
```bash
    [Adldap\Models\ModelNotFoundException]
    No LDAP query results for filter: [(&(objectclass=person)(anr=(uid=user.testing)))] in: [o=intranet.rts]
```
Command: `php artisan adldap:import  --connection "adcare" "(samaccountname=user.testing)"`
```bash
    [Adldap\Models\ModelNotFoundException]
    No LDAP query results for filter: [(&(objectclass=person)(anr=samaccountname=user.testing))] in: [o=intranet.rts]
```
I´m not sure if param `--connection` it´s working, because the basedn it´s the same for both, but if I willfully change the ad connection with incorrect port, I get:
```bash
    [Adldap\Auth\BindException]
    Can't contact LDAP server
```

**Comment by stevebauman** [comment]
Hi @Albvadi, I think this is a bug (or lack of the feature being implemented) in `v3.0.5` using a different connection for importing users (which you're trying to accomplish).

This is my mistake, and will be fixed shortly :).

**Comment by stevebauman** [comment]
Hi @Albvadi, this was actually removed in the new release of `v4.0`. It became too difficult to manage different connections with the import command (due to all the configuration options available).

If you can write a PR or suggest an implementation that would be great :smile:.
